pycbc/waveform/bank.py

```diff
@@ -398,6 +398,10 @@ def end_frequency(self, index):
         """ Return the end frequency of the waveform at the given index value
         """
         from pycbc.waveform.waveform import props
+        try:
+            return self.table[index].f_final
+        except:
+            pass
```

Comment:
I suggest using a `if hasattr` instead of catching the exception, or being more specific about which exception should be ignored.

Reply:
Makes sense, change made.